2024-03-28 09:55:49,304 INFO    StreamThr :5128 [internal.py:wandb_internal():86] W&B internal server running at pid: 5128, started at: 2024-03-28 09:55:49.304442
2024-03-28 09:55:49,314 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status
2024-03-28 09:55:49,324 INFO    WriterThread:5128 [datastore.py:open_for_write():87] open: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\run-coddwpru.wandb
2024-03-28 09:55:49,330 DEBUG   SenderThread:5128 [sender.py:send():378] send: header
2024-03-28 09:55:49,332 DEBUG   SenderThread:5128 [sender.py:send():378] send: run
2024-03-28 09:55:51,810 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: check_version
2024-03-28 09:55:51,811 INFO    SenderThread:5128 [dir_watcher.py:__init__():211] watching files in: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files
2024-03-28 09:55:51,811 INFO    SenderThread:5128 [sender.py:_start_run_threads():1099] run started: coddwpru with start time 1711599949.314132
2024-03-28 09:55:51,811 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: check_version
2024-03-28 09:55:52,223 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: run_start
2024-03-28 09:55:52,257 DEBUG   HandlerThread:5128 [system_info.py:__init__():26] System info init
2024-03-28 09:55:52,257 DEBUG   HandlerThread:5128 [system_info.py:__init__():41] System info init done
2024-03-28 09:55:52,257 INFO    HandlerThread:5128 [system_monitor.py:start():194] Starting system monitor
2024-03-28 09:55:52,257 INFO    SystemMonitor:5128 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-03-28 09:55:52,257 INFO    HandlerThread:5128 [system_monitor.py:probe():214] Collecting system info
2024-03-28 09:55:52,264 INFO    SystemMonitor:5128 [interfaces.py:start():190] Started cpu monitoring
2024-03-28 09:55:52,264 INFO    SystemMonitor:5128 [interfaces.py:start():190] Started disk monitoring
2024-03-28 09:55:52,264 INFO    SystemMonitor:5128 [interfaces.py:start():190] Started gpu monitoring
2024-03-28 09:55:52,264 INFO    SystemMonitor:5128 [interfaces.py:start():190] Started memory monitoring
2024-03-28 09:55:52,278 INFO    SystemMonitor:5128 [interfaces.py:start():190] Started network monitoring
2024-03-28 09:55:52,284 DEBUG   HandlerThread:5128 [system_info.py:probe():150] Probing system
2024-03-28 09:55:52,284 DEBUG   HandlerThread:5128 [gitlib.py:_init_repo():56] git repository is invalid
2024-03-28 09:55:52,284 DEBUG   HandlerThread:5128 [system_info.py:probe():198] Probing system done
2024-03-28 09:55:52,284 DEBUG   HandlerThread:5128 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.13', 'heartbeatAt': '2024-03-28T04:25:52.284361', 'startedAt': '2024-03-28T04:25:49.294240', 'docker': None, 'cuda': None, 'args': ('8000',), 'state': 'running', 'program': 'C:\\Users\\chinm\\Desktop\\FL-FINAL\\MNIST\\server.py', 'codePathLocal': 'server.py', 'codePath': 'server.py', 'host': 'MSI', 'username': 'chinm', 'executable': 'C:\\Users\\chinm\\anaconda3\\New folder\\python.exe', 'cpu_count': 6, 'cpu_count_logical': 12, 'cpu_freq': {'current': 2496.0, 'min': 0.0, 'max': 2496.0}, 'cpu_freq_per_core': [{'current': 2496.0, 'min': 0.0, 'max': 2496.0}], 'disk': {'/': {'total': 458.81444931030273, 'used': 429.2377128601074}}, 'gpu': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'memory_total': 4294967296}], 'memory': {'total': 7.8259735107421875}}
2024-03-28 09:55:52,284 INFO    HandlerThread:5128 [system_monitor.py:probe():224] Finished collecting system info
2024-03-28 09:55:52,284 INFO    HandlerThread:5128 [system_monitor.py:probe():227] Publishing system info
2024-03-28 09:55:52,284 DEBUG   HandlerThread:5128 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-03-28 09:55:52,284 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:55:52,814 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\conda-environment.yaml
2024-03-28 09:55:54,316 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:55:56,340 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:55:58,360 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:00,384 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:02,426 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:04,450 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:06,471 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:07,970 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\conda-environment.yaml
2024-03-28 09:56:08,208 DEBUG   HandlerThread:5128 [system_info.py:_save_conda():219] Saving conda packages done
2024-03-28 09:56:08,209 INFO    HandlerThread:5128 [system_monitor.py:probe():229] Finished publishing system info
2024-03-28 09:56:08,224 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:08,224 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: keepalive
2024-03-28 09:56:08,224 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:08,224 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: keepalive
2024-03-28 09:56:08,224 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:08,224 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: keepalive
2024-03-28 09:56:08,225 DEBUG   SenderThread:5128 [sender.py:send():378] send: files
2024-03-28 09:56:08,225 INFO    SenderThread:5128 [sender.py:_save_file():1365] saving file wandb-metadata.json with policy now
2024-03-28 09:56:08,492 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:08,894 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: python_packages
2024-03-28 09:56:08,894 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: python_packages
2024-03-28 09:56:08,894 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:56:08,894 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:56:08,904 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:56:08,984 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\wandb-metadata.json
2024-03-28 09:56:08,984 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\requirements.txt
2024-03-28 09:56:09,240 DEBUG   SenderThread:5128 [sender.py:send():378] send: telemetry
2024-03-28 09:56:09,538 INFO    wandb-upload_0:5128 [upload_job.py:push():131] Uploaded file C:\Users\chinm\AppData\Local\Temp\tmpx4jwg61ewandb\0k45l99f-wandb-metadata.json
2024-03-28 09:56:09,999 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:56:10,534 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:11,266 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:12,026 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:56:12,549 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:14,570 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:16,303 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:16,584 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:18,606 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:20,639 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:21,349 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:22,128 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\config.yaml
2024-03-28 09:56:22,713 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:23,901 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:56:23,901 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:56:23,901 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:56:24,738 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:26,765 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:27,780 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:28,796 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:30,815 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:32,830 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:32,847 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:34,868 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:36,884 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:37,877 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:38,903 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:56:38,903 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:56:38,905 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:56:38,906 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:40,924 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:42,934 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:43,500 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:44,353 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:56:44,964 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:46,981 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:48,537 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:49,014 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:51,032 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:52,289 DEBUG   SystemMonitor:5128 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-03-28 09:56:52,289 DEBUG   SenderThread:5128 [sender.py:send():378] send: stats
2024-03-28 09:56:53,093 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:53,909 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:56:53,909 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:53,925 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:56:53,925 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:56:55,124 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:57,164 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:56:58,460 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:56:59,189 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:56:59,212 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:01,278 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:03,325 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:04,217 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:05,378 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:07,519 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:08,919 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:57:08,973 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:57:08,979 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:57:09,438 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:09,613 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:11,780 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:13,937 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:14,507 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:16,084 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:18,255 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:19,669 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:20,395 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:22,319 DEBUG   SenderThread:5128 [sender.py:send():378] send: stats
2024-03-28 09:57:22,535 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:24,023 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:57:24,024 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:57:24,032 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:57:24,783 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:25,469 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:26,830 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:28,894 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:30,564 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:30,975 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:33,036 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:35,072 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:36,044 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:57:36,089 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:37,101 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:38,068 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:57:38,926 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:57:39,009 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:57:39,014 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:57:39,144 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:41,175 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:41,471 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:43,206 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:45,234 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:46,544 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:47,351 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:49,428 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:51,490 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:51,628 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:52,355 DEBUG   SenderThread:5128 [sender.py:send():378] send: stats
2024-03-28 09:57:53,716 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:53,933 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:57:54,118 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:57:54,121 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:57:55,958 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:57:56,660 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:57:58,092 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:00,143 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:02,018 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:02,359 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:04,522 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:06,635 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:07,424 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:08,736 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:08,932 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:58:09,117 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:58:09,119 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:58:10,800 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:12,655 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:12,852 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:14,928 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:16,979 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:18,277 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:19,037 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:19,416 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:58:21,094 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:21,443 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:58:22,371 DEBUG   SenderThread:5128 [sender.py:send():378] send: stats
2024-03-28 09:58:23,157 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:23,388 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:23,934 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:58:24,125 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:58:24,127 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:58:25,207 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:27,450 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:28,690 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:29,567 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:31,598 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:33,683 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:33,787 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:35,734 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:37,865 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:39,008 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:39,012 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 09:58:39,137 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 09:58:39,138 DEBUG   SenderThread:5128 [sender.py:send_request():405] send_request: stop_status
2024-03-28 09:58:39,948 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:42,036 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:44,114 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:44,591 DEBUG   HandlerThread:5128 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 09:58:46,130 ERROR   gpu       :5128 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 09:58:46,792 INFO    cpu       :5128 [interfaces.py:monitor():140] Process cpu has exited.
2024-03-28 09:58:46,793 DEBUG   SystemMonitor:5128 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-03-28 09:58:46,793 DEBUG   SystemMonitor:5128 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-03-28 09:58:46,795 DEBUG   SenderThread:5128 [sender.py:send():378] send: stats
2024-03-28 09:58:47,390 INFO    Thread-16 :5128 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\files\output.log
2024-03-28 09:58:48,808 WARNING StreamThr :5128 [internal.py:is_dead():414] Internal process exiting, parent pid 3996 disappeared
2024-03-28 09:58:48,808 ERROR   StreamThr :5128 [internal.py:wandb_internal():152] Internal process shutdown.
2024-03-28 09:58:48,808 INFO    HandlerThread:5128 [handler.py:finish():869] shutting down handler
2024-03-28 09:58:48,808 INFO    HandlerThread:5128 [system_monitor.py:finish():203] Stopping system monitor
2024-03-28 09:58:48,808 INFO    HandlerThread:5128 [interfaces.py:finish():202] Joined cpu monitor
2024-03-28 09:58:48,808 INFO    WriterThread:5128 [datastore.py:close():296] close: C:\Users\chinm\Desktop\FL-FINAL\MNIST\wandb\run-20240328_095549-coddwpru\run-coddwpru.wandb
2024-03-28 09:58:48,808 INFO    HandlerThread:5128 [interfaces.py:finish():202] Joined disk monitor
2024-03-28 09:58:48,809 INFO    HandlerThread:5128 [interfaces.py:finish():202] Joined gpu monitor
2024-03-28 09:58:48,809 INFO    HandlerThread:5128 [interfaces.py:finish():202] Joined memory monitor
2024-03-28 09:58:48,809 INFO    HandlerThread:5128 [interfaces.py:finish():202] Joined network monitor
2024-03-28 09:58:48,819 INFO    SenderThread:5128 [sender.py:finish():1521] shutting down sender
2024-03-28 09:58:49,117 INFO    MainThread:5128 [internal.py:handle_exit():76] Internal process exited
